src/test/java/uk/gov/hmcts/reform/professionalapi/domain/RoleNameTest.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.professionalapi.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class RoleNameTest {
+
+    @Test
+    public void should_hold_values_after_creation() {
+
+        RoleName roleName = new RoleName();
+        roleName.setName("pui-case-manager");

```

Comment:
Enums or public static final strings would be better than local hardcoded references over and over

Reply:
added private final class variables